### PR TITLE
IA-3215 move resize2fs up

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://broadworkbench.atlassian.net/browse/IA-3215

See https://terra-bio.slack.com/archives/C9C4KKU65/p1645061109185549?thread_ts=1644552492.327509&cid=C9C4KKU65

* Move resize2fs up a bit so that resizing happens sooner
* Bump a few dependencies
* Add metrics for http4s client used for Jupyter, RStudio, etc

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/resources/init-resources/gce-init.sh [hunk 1]
```diff
@@ -231,6 +231,15 @@ fi
 
 mount -t ext4 -O discard,defaults /dev/${DISK_DEVICE_ID} ${WORK_DIRECTORY}
 
+# Resize persistent disk if needed.
```
Comment (calbach) [inline http/src/main/resources/init-resources/gce-init.sh]: Adding some logic to that PD integration test which writes a file, then reads it back after re-attach would be great as well.

**Review by calbach — APPROVED** [review]
Thanks

**Comment by Qi77Qi** [comment]
weird config error

```
2022-02-17T21:14:00.339Z [ERR] (cli) /working/configs/leonardo/application.conf.ctmpl: parse: template: :6: function "vault" not defined

FATAL ERROR: Unable to execute consule-template for /working/configs/leonardo/application.conf.ctmpl!
no env file specified
using default consul template path
using default consul config
```

jenkins retest

**Review by gpcarr — APPROVED** [review]
lgtm

**Comment by Qi77Qi** [comment]
same vault error


jenkins retest

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by Qi77Qi** [comment]
leonardo build error

jenkins retest

**Comment by calbach** [comment]
Eager to test this out - let me know if I can help

**Comment by Qi77Qi** [comment]
leonardo-build failed.. jenkins retest

**Comment by Qi77Qi** [comment]
@gpcarr I bumped a few dependencies and added metrics for http4s client used for Jupyter, RStudio, etc as well in this PR....lmk if this still looks good to you

**Comment by Qi77Qi** [comment]
unrelated mysql error

jenkins retest

**Pull request merged**